PROJ-39: Uwb (#33)
SW805F20/report: sections/sprint-1/pozyx.tex

The issue with this approach is that the measurements are not perfect, which might cause the circles to not intersect at exactly one point which will make the positional data seem to jitter.

\subsection{Using UWB}
To find the position of the tags, Pozyx makes use of radio waves. 
Radio waves travel at the speed of light, so by dividing the time of travel between anchors with the speed of light, the distance between them can be found.\\
Because the speed of light is so fast the time measure needs to be very accurate to get the correct distance.
To achieve this the anchors make use of UWB \cite{pozyx-UWB}.
UWB is a technology for transmitting data over a wide bandwidth usually wider than 500 MHz.
The Pozyx sensors use exactly 500 MHz which means they just qualify as a UWB system.

> Because the bandwidth is this wide it makes the wavelength very short and by combining multiple sinusoidal signals with slightly different frequencies the Pozyx tags can create a pulse with a peak which is very narrow and measuring a narrow peak results in a more accurate timing allowing the Pozyx to much more precise than other technologies like Bluetooth and WiFi.

Reviewer: ```suggestion
Because the bandwidth is this wide it makes the wavelength very short, and by combining multiple sinusoidal signals with slightly different frequencies the Pozyx tags can create a pulse with a peak which is very narrow.
Measuring a narrow peak results in a more accurate timing, allowing the Pozyx to be much more precise than other technologies like Bluetooth and WiFi.
```

> But high bandwidth means faster data transfer, which most people would prefer, but if everyone were to use the same frequency the signals would interfere with each other, therefore the use of high-frequency signals is tightly regulated\cite{tait-radio}.

Reviewer: ```suggestion
High bandwidth means faster data transfer, which most people would prefer, but if everyone were to use the same frequency the signals would interfere with each other, therefore the use of high-frequency signals is tightly regulated\cite{tait-radio}.
```

To reduce the amount of interference the regulations say that UWB systems may only transmit at very low power.
This means that a UWB transmission can not travel very far and therefore the chance of it interfering with another system is very low.
Because of this we are not able to build huge playing fields with the limited amount of anchors we have if we also want it to be precise or even playable.

> Building a too big playing field could cause some tags to not reach all anchors and therefor not calculate a position.

Reviewer: ```suggestion
Building a playing field that is too big could cause some tags to not reach all anchors and therefore not calculate a position.
```

> This could be combated somehow by having more anchors but as will be explained in \ref{pozyx:TWR} the slave tags might not be able to communicate with the master tag because the distance between them is too large.

Reviewer: ```suggestion
This could be combated somewhat by having more anchors, but as will be explained in \ref{pozyx:TWR} the slave tags might not be able to communicate with the master tag because the distance between them is too large.
```
